src/plugins/auctionHouseModule/createBid.ts

```diff
@@ -17,12 +47,30 @@ export type CreateBidOperation = Operation<
 >;
 
 export type CreateBidInput = {
+  auctionHouse: AuctionHouse;
+  wallet?: PublicKey | Signer; // Default: identity
+  authority?: PublicKey | Signer; // Default: auctionHouse.authority
+  mintAccount: PublicKey; // Required for checking Metadata
+  tokenAccount?: PublicKey; // Default: ATA
```

Comment:
This attribute has a different usage than listing. We're no longer selling, so the token account is not the ATA of the buyer. Instead, the token account tells us if we're trying to do a private or public bid.

On top of that, we should offer an easier way for the user to tell who they want to buy from. They can either provide a `tokenAccount` or a `seller` address. If they provide the latter, we'll get the token account from the ATA of the `mintAccount` and the `seller` address. If they don't provide any, it's a public bid. All of these use cases should be tested.

```suggestion
  seller?: Option<PublicKey>; // Default: null (i.e. public bid unless tokenAccount is provided)
  tokenAccount?: Option<PublicKey>; // Default: null (i.e. public bid unless seller is provided, in which case, default to the ATA of the seller).
```